src/PythHttpClient.ts

```diff
@@ -0,0 +1,138 @@
+import { AccountInfo, Commitment, Connection, PublicKey } from "@solana/web3.js";
+import { Product, PriceData, parseProductData, parsePriceData, parseBaseData, AccountType } from ".";
+
+const ONES = '1111111111111111111111111111111';
+
+interface PriceRawData {
+    key: PublicKey;
+    account: AccountInfo<Buffer>;
+}
+
+export interface PythHttpClientResult {
+    assetsTypes: string[];
+    productsSymbols: string[];
+    products: Product[];
+    productPrice: Map<string, PriceData>;
```

Comment:
might be nice for the values here to have both the Product and the PriceData. at the moment, there's no easy way to look up the product metadata for a symbol using this interface.

Reply:
added productFromSymbol map